tests/containers/run_container_in_k3s.pm

```diff
@@ -16,8 +16,20 @@ use serial_terminal 'select_serial_terminal';
 use utils;
 use version_utils;
 use publiccloud::utils;
+use containers::utils qw(check_min_runtime_version);
+use Utils::Architectures qw(is_ppc64le);
 use containers::k8s qw(install_k3s uninstall_k3s apply_manifest wait_for_k8s_job_complete find_pods validate_pod_log);
 
+sub prep_pod_yaml {
+    record_info('Prep', 'Generate the yaml from a pod');
+    my $image = "registry.suse.com/bci/bci-busybox:latest";
+    script_retry("podman pull $image", timeout => 300, delay => 60, retry => 3);
+    assert_script_run("podman container create -t --pod new:testing-pod --name container $image");
```

Comment:
```suggestion
    assert_script_run("podman container create -t --pod new:testing-pod $image");
```
I would remove the name here, because it's not needed.

Reply:
@grisu48 Fixed.

Reply:
Thank you!